tests/tailor.js

```diff
@@ -800,4 +801,236 @@ describe('Tailor', () => {
         }).then(done, done);
     });
 
+    describe('without options `maxScriptLinks` and `maxStyleLinks` provided', () => {
+        it('should handle the last fragment-script Header Link only', (done) => {
+            nock('https://fragment')
+                .get('/1').reply(200, 'hello maxScriptLinks default', {
+                    'Link': '<http://link1>; rel="fragment-script", <http://link2>; rel="fragment-script", <http://link3>; rel="fragment-script"'
+                });
+
+            mockTemplate
+                .returns('<fragment src="https://fragment/1"></fragment>');
+
+            getResponse('http://localhost:8080/test').then((response) => {
+                assert.equal(response.body,
+                    '<html><head></head><body>' +
+                    '<script data-pipe>p.start(0, "http://link3", {"id":0})</script>' +
+                    'hello maxScriptLinks default' +
+                    '<script data-pipe>p.end(0, "http://link3", {"id":0})</script>' +
+                    '</body></html>'
+                );
+            }).then(done, done);
+        });
+
+        it('should handle the last stylesheet Header Link only', (done) => {
```

Comment:
I picked the last one to ensure not to introduce a breaking change. However I am also fine with the first one[s] which is more intuitive